common/env.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+/*
+ * This file used to be in v1/src, was moved to common by using the ILambdaFunction interface
+ * in place of importing Function from @aws-cdk/aws-lambda
+ */
+
```

Comment:
This file used to be in v1/src, was moved to common by using the ILambdaFunction interface in place of importing Function from @aws-cdk/aws-lambda